lib/quiltt_connector.dart

```diff
@@ -143,6 +166,9 @@ class _WebViewPage {
         onExitAbort?.call(Event(type: eventType, eventMetadata: eventMetadata));
         _closeWebView();
         break;
+      case 'authenticate':
+        // @todo handle Authenticate
```

Comment:
@sirwolfgang I forgot what were we suppose to do in `authenticate`?

Reply:
This was exposed as a callback for web, to allow hiding of the loading box. I don't think we need to do that in mobile. Mobile is fullscreen, so they are going to get loading screen.

Reply:
Added https://github.com/quiltt/quiltt-connector-flutter-sdk/pull/11/commits/13310f7cce22cf2f2b225ca30cd1c88909682d65